doc/performance-investigations/README.md

```diff
@@ -63,10 +63,18 @@ OUTPUT_DIR=outputs/baseline pipenv run python -m scripts.get_aggregated_summary
 Example output:
 
 ```
-GETs average: 116ms
-POSTs average: 146ms
-All requests average: 131ms
-Total Requests: 204,438
+Percentiles:
+50th: 74ms
+90th: 140ms
+95th: 170ms
+99th: 240ms
+99.9th: 360ms
+100th: 820ms
```

Comment:
I noticed that the 100th percentile was quite a bit lower with the recent changes for weighting. After reading through https://www.circonus.com/2018/11/the-problem-with-percentiles-aggregation-brings-aggravation/ it looks like aggregating percentiles should not usually be done, as it doesn’t work as expected and hides outliers. However, as we don't have the original data, we can't do it properly, and the average percentiles are useful as an indication of performance. I think we should label them as `Percentile Averages` and remove the 100th percentile from the output as I think most people would expect that to be the longest request time.

Reply:
Agreed. Have made the changes.